$NetBSD: patch-ai,v 1.1 2001/03/14 10:29:42 agc Exp $

A very quick and dirty workaround to an autoconf problem.

The GNU configure script gets the HAVE_SYS_DISKLABEL_H wrong on NetBSD
systems after January 2001, since the inclusion of <sys/disklabel.h>
will only compile cleanly if <sys/types.h> has been included
previously, which the configure script doesn't do.  Hence
HAVE_SYS_DISKLABEL_H goes undefined.

For now, make it work on NetBSD.

--- agent/mibgroup/host/hr_disk.c	2001/03/14 10:20:40	1.1
+++ agent/mibgroup/host/hr_disk.c	2001/03/14 10:21:00
@@ -41,7 +41,7 @@
 #if HAVE_LINUX_HDREG_H
 #include <linux/hdreg.h>
 #endif
-#if HAVE_SYS_DISKLABEL_H
+#if defined(__NetBSD__) || HAVE_SYS_DISKLABEL_H
 #define DKTYPENAMES
 #include <sys/disklabel.h>
 #endif
